files/zh-cn/web/api/rtcpeerconnection/index.md

```diff
@@ -1,252 +1,138 @@
-
-    - `"stable"`: 没有 SDP offer/answer 正在被交换，连接仍然处于初始化状态。
-    - `"have-local-offer"`: 这条连接的本地端机器已经本地应用了一个 SDP offer。
-    - `"have-remote-offer"`: 这条连接的远端机器已经本地应用了一个 SDP offer。
-    - `"have-local-pranswer"`: 一个来自远端的 SDP offer 已经被应用，同时一个 SDP pranswer 在本地被应用。
-    - `"have-remote-pranswer"`：一个本地的 SDP offer 被应用，同时一个 SDP pranswer 在远端被应用。
-    - `"closed"`: 连接被关闭。
-
-### 事件处理器
-
-- {{domxref("RTCPeerConnection.onaddstream")}}
-  - : 是收到 `addstream` 事件时调用的事件处理器。Such an event is 当{{domxref("MediaStream")}} 被远端机器添加到这条连接时，该事件会被触发。当调用{{domxref("RTCPeerConnection.setRemoteDescription()")}}方法时，这个事件就会被立即触发，它不会等待 SDP 协商的结果。
-- {{domxref("RTCPeerConnection.ondatachannel")}}
-  - : 是收到 `datachannel` 事件时调用的事件处理器。当一个 {{domxref("RTCDataChannel")}} 被添加到连接时，这个事件被触发。
-- {{domxref("RTCPeerConnection.onicecandidate")}}
-  - : 是收到 `icecandidate` 事件时调用的事件处理器.。当一个 {{domxref("RTCICECandidate")}} 对象被添加时，这个事件被触发。
-- {{domxref("RTCPeerConnection.oniceconnectionstatechange")}}
-  - : 是收到 `iceconnectionstatechange` 事件时调用的事件处理器。当{{domxref("RTCPeerConnection.iceConnectionState", "iceConnectionState")}} 改变时，这个事件被触发。
-- {{domxref("RTCPeerConnection.onidentityresult")}}
-  - : 是收到 `identityresult` 事件时调用的事件处理器。当通过{{domxref("RTCPeerConnection.getIdentityAssertion()", "getIdentityAssertion()")}}生成身份断言，或在生成一个 answer 或一个 offer 的过程中，这个事件被触发。
-- {{domxref("RTCPeerConnection.onidpassertionerror")}}
-  - : 是收到 `idpassertionerror` 事件时调用的事件处理器。当生成一个身份断言时，如果关联的身份提供者（idP）遇到一个错误，这个事件就会被触发。
-- {{domxref("RTCPeerConnection.onidpvalidationerror")}}
-  - : 是收到 `idpvalidationerror` 事件时调用的事件处理器。当检查 一个身份断言时，如果关联的身份提供者（idP）遇到一个错误，这个事件就会被触发。
-- {{domxref("RTCPeerConnection.onnegotiationneeded")}}
-  - : 是收到 `negotiationneeded` 事件时调用的事件处理器，浏览器发送该事件以告知在将来某一时刻需要协商。
-- {{domxref("RTCPeerConnection.onpeeridentity")}}
-  - : 是收到 `peeridentity` 事件时调用的事件处理器，当一条连接的 peer identify 被设置以及校验后，该事件被触发
-- {{domxref("RTCPeerConnection.onremovestream")}}
-  - : 是收到 `removestream` 事件时调用的事件处理器，当一条{{domxref("MediaStream")}} 从连接上移除时，该事件被触发。
-- {{domxref("RTCPeerConnection.onsignalingstatechange")}}
-  - : 是收到 `signalingstatechange` 事件时调用的事件处理器，当{{domxref("RTCPeerConnection.signalingState", "signalingState")}}的值发生改变时，该事件被触发。
+{{InheritanceDiagram}}
 
 ## 构造函数
 
-- {{domxref("RTCPeerConnection.RTCPeerConnection", "RTCPeerConnection()")}}
-  - : RTCPeerConnection 的初始化函数，通过 new RTCPeerConnection() 初始化一个 RTCPeerConnection 实例。
+- {{DOMxRef("RTCPeerConnection.RTCPeerConnection", "RTCPeerConnection()")}}
+  - : 返回一个新的 `RTCPeerConnection` 实例，表示本地设备和远程对等端之间的连接。
+
+## 实例属性
+
+_也从 {{domxref("EventTarget")}} 继承属性。_
+
+- {{DOMxRef("RTCPeerConnection.canTrickleIceCandidates", "canTrickleIceCandidates")}} {{ReadOnlyInline}}
+  - : 返回一个布尔值，指示远程对等方是否可以接受[trickled ICE candidates](https://datatracker.ietf.org/doc/html/draft-ietf-mmusic-trickle-ice) 候选者。
```

Comment:
I'm not sure if this term needs to be translated, pls update it if needed.
```suggestion
  - : 返回一个布尔值，指示远程对等方是否可以接受[trickled ICE candidates](https://datatracker.ietf.org/doc/html/draft-ietf-mmusic-trickle-ice)。
```

Reply:
ref: https://github.com/mdn/translated-content/pull/18143#discussion_r1477069065